src/api/src/services/BinaryFlashingStrategy/index.ts

```diff
@@ -1,50 +1,50 @@
 /* eslint-disable no-bitwise */
-import { Service } from 'typedi';
+import fs from 'fs';
 import { PubSubEngine } from 'graphql-subscriptions';
 import * as os from 'os';
-import semver from 'semver';
 import path from 'path';
-import fs from 'fs';
-import UserDefine from '../../models/UserDefine';
-import FirmwareSource from '../../models/enum/FirmwareSource';
+import semver from 'semver';
+import { Service } from 'typedi';
+import UserDefinesTxtFactory from '../../factories/UserDefinesTxtFactory';
+import TargetArgs from '../../graphql/args/Target';
+import GitRepository from '../../graphql/inputs/GitRepositoryInput';
+import BuildFlashFirmwareResult from '../../graphql/objects/BuildFlashFirmwareResult';
+import FirmwareBuilder from '../../library/FirmwareBuilder';
+import UserDefineKey from '../../library/FirmwareBuilder/Enum/UserDefineKey';
+import {
+  GitFirmwareDownloader,
+  findGitExecutable,
+} from '../../library/FirmwareDownloader';
 import Mutex from '../../library/Mutex';
+import Platformio from '../../library/Platformio';
+import { LoggerService } from '../../logger';
+import Device from '../../models/Device';
+import UserDefine from '../../models/UserDefine';
 import BuildFirmwareErrorType from '../../models/enum/BuildFirmwareErrorType';
-import PubSubTopic from '../../pubsub/enum/PubSubTopic';
+import BuildJobType from '../../models/enum/BuildJobType';
 import BuildProgressNotificationType from '../../models/enum/BuildProgressNotificationType';
 import BuildFirmwareStep from '../../models/enum/FirmwareBuildStep';
-import { LoggerService } from '../../logger';
-import UserDefineKey from '../../library/FirmwareBuilder/Enum/UserDefineKey';
+import FirmwareSource from '../../models/enum/FirmwareSource';
+import UserDefinesMode from '../../models/enum/UserDefinesMode';
+import PubSubTopic from '../../pubsub/enum/PubSubTopic';
 import { BuildFlashFirmwareParams } from '../FlashingStrategyLocator/BuildFlashFirmwareParams';
-import {
-  createBinaryCopyWithCanonicalName,
-  removeDirectoryContents,
-} from '../FlashingStrategyLocator/artefacts';
+import { FirmwareVersionData } from '../FlashingStrategyLocator/FirmwareVersionData';
 import {
   FlashingStrategy,
   IsCompatibleArgs,
 } from '../FlashingStrategyLocator/FlashingStrategy';
-import TargetArgs from '../../graphql/args/Target';
-import GitRepository from '../../graphql/inputs/GitRepositoryInput';
-import Device from '../../models/Device';
-import { UserDefineFilters } from '../UserDefinesLoader';
-import BuildJobType from '../../models/enum/BuildJobType';
-import BuildFlashFirmwareResult from '../../graphql/objects/BuildFlashFirmwareResult';
 import {
-  findGitExecutable,
-  GitFirmwareDownloader,
-} from '../../library/FirmwareDownloader';
-import DeviceDescriptionsLoader from './DeviceDescriptionsLoader';
-import { FirmwareVersionData } from '../FlashingStrategyLocator/FirmwareVersionData';
-import Platformio from '../../library/Platformio';
```

Comment:
I think this imports reordering is coming from your local environment, not from expresslrs configurator eslint rules. But if you want you could add eslint rule to order imports in a separate PR. I like the idea of ordered imports :)

Reply:
Backed out the change.